packages/fxa-settings/src/pages/CannotCreateAccount/index.tsx

```diff
@@ -8,19 +8,19 @@ import { RouteComponentProps } from '@reach/router';
 import AppLayout from '../../components/AppLayout';
 import CardHeader from '../../components/CardHeader';
 import { FtlMsg } from 'fxa-react/lib/utils';
-import { logPageViewEvent } from '../../lib/metrics';
+import { usePageViewEvent } from '../../lib/metrics';
```

Comment:
it looks like we aren't using a dependencies array where we use `usePageViewEvent` - is there an advantage to using it?

Reply:
My question was answered by your above comment!! Thank you!